jans-cedarling/bindings/cedarling_wasm/README.md

```
@@ -335,3 +477,136 @@ const BOOTSTRAP_CONFIG = {
+```javascript
+// Get data by key
+const value = await cedarling.get_data_ctx("user:123");
+if (value) {
+  console.log(`User roles: ${value.role}`);
+}
+```
+
+### Get Data Entry with Metadata
+
+Get a data entry with full metadata including creation time, expiration, access count, and type:
+```javascript
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)